src/reaction/first_order_reaction_base.hh

```diff
@@ -111,7 +111,7 @@ protected:
     arma::mat molar_matrix_;      ///< Diagonal matrix with molar masses of substances.
     arma::mat molar_mat_inverse_; ///< Inverse of @p molar_matrix_.
 
-    std::shared_ptr<LinearODESolverBase> linear_ode_solver_;
+    std::shared_ptr<PadeApproximant> linear_ode_solver_;
```

Comment:
Now, possibly we may use PadeApproximant directly, but I agree to leave it like this for a while.  Possibly we shall deal with some chemical reactions,  ...